client/src/app/project-registration/project-registration-form/project-registration-form.component.html

```diff
@@ -1,20 +1,33 @@
 <h1 class="vf-text vf-text-heading--2">{{title}}</h1>
-<p class="vf-text-body vf-text-body--3 vf-u-text-color--grey">
-  {{subtitle}}
-</p>
 <br/>
 <br/>
 
+<mat-tab-group>
+  <mat-tab label="1. Project">
```

Comment:
the labels are duplicate. See project.compoment

Reply:
We will resolve this in [#225](https://app.zenhub.com/workspaces/dcp-ingest-product-development-5f71ca62a3cb47326bdc1b5c/issues/ebi-ait/dcp-ingest-central/225)